docs/source/install.rst

```diff
@@ -133,3 +133,7 @@ super_prove
 Avy
 ^^^
   https://arieg.bitbucket.io/avy/
+
+rIC3
+^^^
+  https://github.com/gipsyh/rIC3/
```

Comment:
Could you add a note that this needs at least `v1.2.1` of rIC3, given that the HWMCC24 version etc don't have the `--sby` flag.